core/validation/pre_transformation_validate.py

```diff
@@ -18,33 +25,44 @@ def pre_transformation_validations(
 ):
     """Performs pre-transformation validations based on the provided schema.
 
+    The submission form structures of TF Round 1 & Round 2 do not permit pre-transformation validations
+    as they have already been transformed.
+    Authorisation was introduced in TF Round 4 in order to ensure that users did not submit for local authorities
+    they do not belong to, and does not occur for previous TF rounds.
+    wrong_input_validation occurs first as we cannot authorise user input if it is invalid.
+    conflicting_input_validation occurs next as we cannot authorise user input if it is contradictory.
+
     :param workbook: A dictionary where keys are sheet names and values are pandas
                      DataFrames representing each sheet in the submission.
     :param reporting_round: The reporting round for which the validations are performed.
-    :param auth: A dictionary containing authorization information.
+    :param auth: A dictionary containing authorisation information.
     :raises ValidationError: If any of the validation functions catch a validation error.
     """
     if reporting_round in [1, 2]:
-        return  # do not do pre-transformation validation on historical data
+        return
 
     schema = REPORTING_ROUND_TO_PRE_TRANSFORMATION_SCHEMA[reporting_round]
 
-    wrong_input_validation(workbook, schema)  # first, check if valid
+    wrong_input_validation(workbook, schema)
     conflicting_input_validation(workbook, schema)
 
     if reporting_round == 4 and auth:
         authorisation_validation(workbook, auth, schema)
```

Comment:
Just thinking, instead of passing reporting round etc. We could refactor this so the schema determines the checks. i.e. if the schema has wrong input checks then you do that, but if it didn't include any authorisation then don't execute that. 

Means this is even more schema driven and would help with extendibility to get rid of the concept of reporting rounds in the code itself

I realise this is a bit out of scope but may be worth doing for clarity.. We can then document about the difference in validation functionality where the schema is defined, rather than here - which should really be round/fund context free

Reply:
We can do this with:

```
    if schema.get("authorisation_checks"):
        authorisation_validation(workbook, auth, schema)
```

If the schema specifies the functions that are to be checked then we will have to use a string representation of each function in the schema (to avoid a circular import), and map these back to a set of functions via a dictionary somewhere else in the code. 

The above code snippet is a bit easier and means we escape reliance on a reporting round to determine if ```authorisation_validation``` is called. If there ends up being a ballooning of pre-transformation validation functions then this can be revisited, but for now I think it's good enough?

Reply:
We still rely on `reporting_round` above but we'll refactor that out later. This change looks fine for now. 

Although you've removed the check on whether `auth` is truthy, which has broken ingest. 

We should still check `auth` exists inline with the old code. I'd recommend adding that back and ideally adding a test to cover this scenario as it should have made a test fail.

Reply:
Nice spot - I overlooked the removal/importance on the check on whether ```auth``` is truthy. Added an extra test case in to account for that.

Reply:
nice 👍